spec/features/admin/admin_creates_new_admin_account_spec.rb

```diff
@@ -2,20 +2,18 @@
 
 feature "admin creates new admin account" do
   scenario "successfully" do
-    skip
-    user = create(:user, admin: true)
+    user = create(:user, :admin)
```

Comment:
A suggestion to change this local variable to be called `admin`

Reply:
Sorry, missed this one, thanks - done in sha: db0a541